src/views/Pipeline.vue

```diff
@@ -334,8 +356,12 @@ export default defineComponent({
       getCurrentEComStore:'user/getCurrentEComStore',
       isPendingJobsScrollable: 'job/isPendingJobsScrollable',
       isRunningJobsScrollable: 'job/isRunningJobsScrollable',
-      isHistoryJobsScrollable: 'job/isHistoryJobsScrollable'
-    })
+      isHistoryJobsScrollable: 'job/isHistoryJobsScrollable',
+      getPinnedJobs: 'user/getPinnedJobs'
+    }),
+    pinnedJobs() {
+      return (this as any).getPinnedJobs?.searchPrefValue ? Object.keys((this as any).getPinnedJobs?.searchPrefValue)?.filter((pref: any) => (this as any).getPinnedJobs?.searchPrefValue[pref]) : [];
```

Comment:
I think this should not be needed. We should implement getter as per the requirement

Reply:
Sir, we only need these job's enums to show in UI we are not performing any action on it, so I think we should use computed property to do this instead of using getter.